frontend/src/views/web/partials/sim-header.html

```diff
@@ -14,6 +14,24 @@
                      <div class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" ng-click="logout()">Logout</div>
                 </li>
             </ul>
+      <ul class="side-nav" id="mobile-demo">
+        <li>
+          <div class="userView">
+            <div class="background"></div>
+            <a><span class="white-text name">EvalAI</span></a>
+          </div>
+        </li>
+        <li><a class="waves-effect waves-dark" ui-sref="home" ui-sref-active="active"><i class="fa fa-home"></i>Home</a></li>
+        <li ng-hide="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="auth.signup" ui-sref-active="active"><i class="fa fa-sign-out"></i>Sign Up</a></li>
+        <li ng-hide="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="auth.login" ui-sref-active="active"><i class="fa fa-sign-in"></i>Log In</a></li>
+        <!-- show after authenticate user -->
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.dashboard" ui-sref-active="active"><i class="fa fa-thumb-tack"></i>Dashboard</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.challenge-main" ui-sref-active="active"><i class="fa fa-fire"></i>All Challenges</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.teams" ui-sref-active="active"><i class="fa fa-users"></i>Participant Teams</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="web.challenge-host-teams" ui-sref-active="active"><i class="fa fa-paper-plane"></i>Create Challenge</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark sim-header-link" ui-sref="auth.signup" ui-sref-active="active"><i class="fa fa-user"></i>Hi {{web.name}}!</a></li>
+      </ul>
+
         </div>
     </nav>
 </div>
```

Comment:
@Obtin Yes, you are right, The code works perfectly! I want you to add the missing fields in the navigation of `Home` page. Please see the two screenshots.
1. Home Page Menu
![timeline10](https://user-images.githubusercontent.com/12206047/35700710-c678507a-07b9-11e8-85df-bcf45ff3389b.jpg)
2. Dashboard Menu
![timeline11](https://user-images.githubusercontent.com/12206047/35700741-d86e9bfe-07b9-11e8-83e2-6e435d91b89b.jpg)